specification/batch/resource-manager/Microsoft.Batch/stable/2019-08-01/BatchManagement.json

```diff
@@ -730,7 +730,7 @@
             "$ref": "./examples/ApplicationPackageCreate.json"
           }
         },
-        "description": "Creates an application package record.",
+        "description": "Creates an application package record. The record contains the SAS where the package should be uploaded to.  Once it is uploaded the `ApplicationPackage` needs to be activated using `ApplicationPackageActive` before it can be used.",
```

Comment:
The term `record` seems unnecessary to me and doesn't really have a meaning for customers.

Reply:
"This record refers to a location in Azure Storage where the package should be uploaded. This API returns an Azure Storage Blob URL with Shared Access Signature which can be used to perform the Application upload"